### PR TITLE
Add AsyncSeq.mapAsyncUnorderedParallel

### PR DESCRIPTION
**Pull request by Xyncgas**
I implemented mapAsyncUnorderedParallel
Can someone please help me to add it into the AsyncSeq library we are using

### CODE REVIEW

**Comment by Xyncgas** [comment]
My first contribution

**Comment by Xyncgas** [comment]
#155

File: Feature/Library.fs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿[<AutoOpen>]
+module ChannelExtention
+    open System.Threading
+    open System.Threading.Channels
+    open FSharp.Control
+
+    open LanguagePrimitives
```
Comment (dsyme) [inline Feature/Library.fs:7]: This shouldn't be needed?

Comment (dsyme) [reply Feature/Library.fs:7]: Usually use `LanguagePrimitives.GenerixZero`

File: Helper/CreateChannel.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Channels;
+namespace Helper
```
Comment (dsyme) [inline Helper/CreateChannel.cs:2]: Any reason why this helper can't be written in F#?

**Comment by dsyme** [comment]
@Xyncgas I left some initial comments - we'd need tests too.  Any reason this doesn't go direct into the main library?

**Comment by Xyncgas** [comment]
> @Xyncgas I left some initial comments - we'd need tests too. Any reason this doesn't go direct into the main library?

I didn't want to put stuff into the folder but created a separate one because I didn't know what to do

**Comment by dsyme** [comment]
@abelbraaksma Should we close this old one out?

**Comment by Xyncgas** [comment]
I'm willing to create a good solution for adding the method as a part of the library

**Pull request closed without merging**